### PR TITLE
frontend: enabling tokens now use toggles on manage accounts

### PR DESCRIPTION
**Pull request by thisconnect**
- added small lines to visualize their close relation to the
  parent ethereum account
- commented out but kept the toggle favor account feature
- linked the coin accounts to their account-overview, but
  have not id for tokens to correctly link to

### CODE REVIEW

**Comment by thisconnect** [comment]
I'll add show/hide token button in this PR with another commit

File: frontends/web/src/routes/settings/manage-accounts.tsx [hunk 1]
```diff
@@ -132,20 +163,27 @@ class ManageAccounts extends Component<Props, State> {
         }
         return Object.entries(this.erc20TokenCodes)
             .map(([tokenCode, name]) => {
-                const active = activeTokens && activeTokens.includes(tokenCode);
+                const active = !!activeTokens && activeTokens.includes(tokenCode);
```
Comment (benma) [inline frontends/web/src/routes/settings/manage-accounts.tsx:166]: are the !! really needed? i didn't see typescript complain about sth like this yet.

File: frontends/web/src/routes/settings/manage-accounts.tsx [hunk 2]
```diff
@@ -132,20 +163,27 @@ class ManageAccounts extends Component<Props, State> {
...
-                        <Logo
-                            active={active}
-                            alt={name}
-                            className={style.tokenIcon}
-                            coinCode={tokenCode}
-                            stacked />
-                        <span className="flex-1 p-left-quarter">
-                            {name} ({tokenCode})
-                        </span>
+                        <div
```
Comment (benma) [inline frontends/web/src/routes/settings/manage-accounts.tsx:170]: why not `<A href...`?

The link is underlined but it also needs a hand/finger mouse pointer.

**Review by benma — APPROVED** [review]
looks nice!

File: frontends/web/src/routes/settings/manage-accounts.tsx (hunk 1 above)
Comment (thisconnect) [reply frontends/web/src/routes/settings/manage-accounts.tsx:166]: IAccount says activeTokens can be undefined/missing is that correct?

https://github.com/digitalbitbox/bitbox-wallet-app/blob/staging-multiaccounts/frontends/web/src/api/account.ts#L39

Screen Shot 2021-05-19 at 12 02 02 PM

The Toggle component wants a boolean, so Typescript complains, could either:
- keep as is
- update IAccounts interface (if backend always returns at least an empty array)
- move `!!` to Toggle, but I think `active ` should be a boolean and never undefined.

File: frontends/web/src/routes/settings/manage-accounts.tsx (hunk 2 above)
Comment (thisconnect) [reply frontends/web/src/routes/settings/manage-accounts.tsx:176]: Inactive tokens shouldn't be linked, so we won't always have a link, I'd prefer at the moment to keep using onClick

File: frontends/web/src/routes/settings/manage-accounts.tsx [hunk 3]
```diff
@@ -157,13 +157,19 @@ class ManageAccounts extends Component<Props, State> {
...
         }
         return Object.entries(this.erc20TokenCodes)
             .map(([tokenCode, name]) => {
                 const active = activeTokens !== undefined && activeTokens.includes(tokenCode);
+                const accountCode = accounts
+                    .filter(account => account.coinCode === tokenCode);
+                    // .filter or .find by my ETH account somehow, need ETH parent account
+                if (accountCode.length > 1) {
+                    console.log(accountCode);
+                }
```
Comment (thisconnect) [inline frontends/web/src/routes/settings/manage-accounts.tsx:172]: @benma as discussed, we'd at least need the parent ETH account or similar

**Comment by thisconnect** [comment]
LGTM

**Pull request merged**